src/Infrastructure/Common/DTO/PaginationDTO.ts

```diff
@@ -1,7 +1,9 @@
+import { Type } from 'class-transformer';
 import { IsNumber, IsOptional, Max, Min } from 'class-validator';
 
 export class PaginationDTO {
   @IsOptional()
+  @Type(() => Number)
   @Min(1)
   @Max(10000)
   @IsNumber()
```

Comment:
Peut-être que si on coerce le type dès le début (je crois que les décorateurs sont appliqués de bas en haut) ça pourrait marcher

```suggestion
  @Min(1)
  @Max(10000)
  @IsNumber()
  @Type(() => Number)
```

Reply:
Top ! merci